cfme/containers/container.py

```diff
@@ -93,6 +44,34 @@ def is_displayed(self):
         return self.summary.is_displayed
 
 
+class ContainerDetailsView(ContainerObjectDetailsBaseView):
+    pass
+
+
+class Container(WidgetasticTaggable, Navigatable):
```

Comment:
**REQUIRED**
Please rewrite this according the latest Collections/Entity Model version.

Reply:
We agreed that first we do the WT conversions and then converting to collection in another PR - right @psav ?